WORKSPACE

```diff
@@ -454,14 +450,13 @@ haskell_register_ghc_nixpkgs(
         "-Wwarn",
     ],
     repositories = dev_env_nix_repos,
-    static_runtime = True,
-    version = "8.6.5",
+    version = "8.10.3",
```

Comment:
static GHC appears to segfault on macos. Given that GHC upgrades have already been fairly painful, not deviating from the default of having a shared build seems like a better choice. We benchmarked performance improvements and while this did seem slightly slower (22min vs 19min) that was on a fully cleaned build (usually you won’t refetch dependencies) and it also had more cache misses so the 3 minutes are probably a bit less.

Reply:
Is there a GHC ticket on the segfault?

Reply:
Only the two tickets I linked to a few days ago https://gitlab.haskell.org/ghc/ghc/-/issues/18181 https://gitlab.haskell.org/ghc/ghc/-/issues/19021. I didn’t yet get around to reproducing it on a simpler example and opening a proper issue for it. (not having direct access to a mac node makes this a bit annoying).